app/kumactl/cmd/install/testdata/install-control-plane.cni-enabled.golden.yaml

```diff
@@ -1995,6 +1995,7 @@ spec:
   selector:
     matchLabels:
       app: kuma-cni
+      parent.app: "kuma-cni"
```

Comment:
What's is this for?

Reply:
This is how I detect the CNI app. We decided with @bartsmykla to go with a label, I'm open to suggestions on how to name this. Also bear in mind that it must also work with `KM` and most of the labels are parametrised with `kuma.name` variable.

Reply:
why isn't app enough?

Reply:
App changes between `KM` and `Kuma` ([here](https://github.com/slonka/kuma/blob/056425695017156d37bf37e479882f7f48233009/deployments/charts/kuma/templates/_helpers.tpl#L149)). If there is an easy way to forward this variable to the taint controller then fine, otherwise I'd prefer something that is the same between `KM` / `Kuma`

Reply:
I'd go with the env var to be the value of this instead of  a bool as explained later

Reply:
I thought we only talked about removing `parent.app` and not getting rid of the whole `experimental.cni` boolean. I could base the `experimental.cni` logic on `KUMA_CNI_APP` being an empty string but it sounds a bit hacky. Are we sure we want to that or I just misunderstood this.

Reply:
I applied the changes with the `KUMA_CNI_APP` so I'm closing this conversation, feel free to open it up again / open an issue for this.